package.json

```diff
@@ -23,7 +23,7 @@
   "readmeFilename": "README.md",
   "dependencies": {
     "axios": "^0.21.1",
-    "eclipsefdn-solstice-assets": "0.0.143",
+    "eclipsefdn-solstice-assets": "^0.0.147",
```

Comment:
We should NOT add ^ here, as we have no minor/major releases. This would cause builds to automatically update if install is ever used over CI (explicit versions are important to ensure regressions don't occur)

Reply:
Sounds good, updated.